go.mod

```diff
@@ -4,11 +4,11 @@ go 1.17
 
 require (
 	github.com/google/addlicense v1.0.0
-	github.com/google/go-cmp v0.5.5
+	github.com/google/go-cmp v0.5.6
 	github.com/onsi/ginkgo v1.14.0
-	github.com/onsi/gomega v1.10.1
-	github.com/prometheus/common v0.31.0
-	github.com/prometheus/prometheus v2.5.0+incompatible
+	github.com/onsi/gomega v1.10.3
```

Comment:
This is used only for testing so we shouldn't need to update it.

Reply:
done